zombie-bite/README.md

```diff
@@ -0,0 +1,45 @@
+# Zombie-bite
+
+## Intro
+
+`zombie-bite` is an cli tool that allow you to _fork and spawn_ live networks (e.g polkadot/kusama) keeping the _live state_ with the needed customizations in order to make the new chain/s keep progressing.
+
+### Instruction to spawn Polkadot(with sudo)/AH
+
+ - Install `zombie-bite`
+
+   ```
+   cargo install --git https://github.com/pepoviola/zombie-bite --bin zombie-bite
+   ```
+
+- Patch and compile polkadot runtime to add sudo
+
+Apply the patch `polkadot_sudo.patch` on top of the polkadot code in `runtimes` directory and then compile the runtime.
+
+```
+cd ../runtimes && cargo build --release -p polkadot-runtime && cd -
+```
+
+- Compile the needed binaries (from `polkadot-sdk-doppelganger`)
```

Comment:
This could also be a script in the justfile.
I haven't had a chance to check the changes on this branch, are they something we could merge in the sdk?

Reply:
Hey, no. We need to keep in separate branch since we are prepend the blockImport pipeline to override the state with sync warp.
Thx!